roles/logstash/tasks/main.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: create configuration directory
+  sudo: yes
+  file:
+    path: /etc/logstash.d
+    state: directory
+    mode: 0755
+  tags:
+    - logstash
+
+- name: configure logstash
+  sudo: yes
+  template:
+    src: logstash.conf.j2
+    dest: "{{ logstash_config_dir }}/logstash.conf"
+  tags:
+    - logstash
+
+- name: destroy old logspout container
+  when: logspout_rebuild_container|bool
+  docker:
+    name: logspout
+    image: "{{ logspout_image }}"
+    state: absent
+  tags:
+    - logspout
+
+- name: run logspout container
+  docker:
+    name: logspout
+    image: "{{ logspout_image }}"
+    state: started
+    restart_policy: "{{ logspout_restart_policy }}"
+    net: "{{ logspout_net }}"
+    ports:
+    - "8000:8000"
+    env:
+      ROUTE_URIS: logstash://{{ logstash_syslog_host }}:5000
+      LOGSPOUT: ignore
+    volumes:
+    - "/var/lib/docker/:/var/run/docker.sock:ro"
```

Comment:
If we want this to join the weave net and for consitency we could set this at the variables level like:
``` logsoupt_docker_host: {{ docker_host }} ```